packages/dev-console-app/tests/css-transform.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-undef
```

Comment:
Why did this change to js?

Reply:
It's my quick way to avoid obeying tsconfig rules, causing `modules.export` to error on testing. Because we're using `isolatedModules` flag, we'll need to do `export {}`, but that causes jest to fail. Only way I'd get around it is to create a new `tsconfig.test.json`, but I thought this fix will suffice since our testing is so simple atm.